test-files/cases/hoot-rnd/network/conflicts/highway-002/Expected.osm

```diff
@@ -105,6 +94,9 @@
         <nd ref="-1"/>
         <nd ref="-2"/>
         <nd ref="-3"/>
+        <nd ref="-4"/>
+        <nd ref="-5"/>
+        <nd ref="-6"/>
```

Comment:
This is a pretty typical example of the changes where a way is removed and the nodes are added to another.